src/views/SaveCredential/SaveCredential.tsx

```diff
@@ -32,21 +32,20 @@ export function SaveCredential(): JSX.Element | null {
 
   const { claimHash } = usePopupData<IAttestation>();
 
-  const [credential, setCredential] = useState<Credential | null>(null);
+  const credentials = useIdentityCredentials();
+
+  const [credential, setCredential] = useState<Credential>();
 
   useSaveCredential(credential);
 
   useEffect(() => {
-    (async () => {
-      try {
-        const savedCredential = await getCredential(claimHash);
-        setCredential({ ...savedCredential, status: 'attested' });
-      } catch (error) {
-        console.error(error);
-        // TODO: decide on the interface for an unknown credential
-      }
-    })();
-  }, [claimHash]);
+    const credential = credentials.find(
+      (credential) => credential.request.rootHash === claimHash,
+    );
+    if (credential) {
```

Comment:
I’d like to see here at least a TODO for the case when the credential was not found.

Reply:
Oh yeah I removed that TODO by mistake